getting-started/local-node/setting-up-a-node.md

```diff
@@ -127,6 +139,25 @@ With Polkadot JS Apps connected, you will see the standalone Moonbeam node produ
 
 ## Querying Account State
 
-With the release of [Moonbase Alpha v3](https://www.purestake.com/news/moonbeam-network-upgrades-account-structure-to-match-ethereum/), Moonbeam now works under a single account format, which is the Ethereum-styled H160 and is now also supported in Polkadot JS Apps. To check the balance of an address, you can simply import your account to the Accounts tab. You can find more information in [this site](/learn/unified-accounts/).
-
+With the release of [Moonbase Alpha v3](https://www.purestake.com/news/moonbeam-network-upgrades-account-structure-to-match-ethereum/), Moonbeam now works under a single account format, which is the Ethereum-styled H160 and is now also supported in Polkadot JS Apps. To check the balance of an address, you can simply import your account to the Accounts tab. You can find more information in the [Unified Accounts](/learn/unified-accounts/) section.
+ 
 Nevertheless, leveraging the Ethereum full RPC capabilities of Moonbeam, you can use [MetaMask](/getting-started/local-node/using-metamask/) to check the balance of that address as well. In addition, you can also use other development tools, such as [Remix](/getting-started/local-node/using-remix/) and [Truffle](/getting-started/local-node/using-truffle/).
+
+## Common Flags and Options
+
+Flags:
+
+- `--dev` - Specifies the development chain
+- `--tmp` - Runs a temporary node in which all of the configuration will be deleted at the end of the process
+- `--rpc-external` - Listen to all RPC interfaces
+- `--ws-external` - Listen to all Websocket interfaces
+
+Options:
+
+- `--sealing` - When blocks should be sealed in the dev service. Options are "instant", "manual", or timer interval in milliseconds [default: instant]
+
+## Pre-funded Development Account
+
+Your standalone build comes with one pre-funded account for development:
+
```

Comment:
We should also mention the moonbeam --help flag. There is a lot to learn there.

Reply:
The commands for getting to the `--help` menu are slightly different when using docker. So since this tutorial is kind of split up by docker or compiling binaries, I put the commands to get to the `--help` menu in their corresponding sections. 

Should I list both of the commands here too?